Adding the Dockerfile that could run onnxruntime server and corresponding instruments (#62)
NonStatic2014/onnxruntime: dockerfiles/README.md

1. Build the docker image from the Dockerfile in this repository.
  ```

>   # If you have a Linux machine, preface this command with "sudo"

Reviewer: You don't have to delete this line.

  
  docker run -it onnxruntime-ngraph
  ```
## ONNX Runtime Server (Preview)
#### Linux 16.04

1. Build the docker image from the directory that contains our server "onnxruntime_server"
  ```
  docker build -t {docker_image_name} .
  ```
  
2. Run the ONNXRuntime server in Docker image

  ```
  docker run -it -v {localModelAbsoluteFolder}:{dockerModelAbsoluteFolder} -e MODEL_ABSOLUTE_PATH={dockerModelAbsolutePath} -p {your_local_port}:8001 {imageName}
  ```
3. Send the request to server

>  - Send the request to the docker through the binding local port:

Reviewer: 1) You don't have to use "- " at beginning. 
2) You may want to have a link to the full usage doc here. 
```suggestion
 Send the request to the docker through the binding local port. Here is the full [usage document](https://github.com/Microsoft/onnxruntime/blob/master/docs/ONNX_Runtime_Server_Usage.md).
```

Author: fixed

  
  docker run -it onnxruntime-ngraph
  ```
## ONNX Runtime Server (Preview)
#### Linux 16.04

> 1. Build the docker image from the directory that contains our server "onnxruntime_server"

Reviewer 2: ```suggestion
1. Build the docker image from the root directory of ONNXRuntime.
```

Author: As currently we copy the onnxruntime_server directly from the current folder into the docker. If we build the docker from the root directory, then we need to specify the relative path to the server which might changed due to our config is set to Debug/Release/ReleaseWithDebug info. For this reason I prefer to build it from the directory contains the server instead of the root folder.

Reviewer 2: Ah my misunderstanding then

  ```

>   

Reviewer: Remove the empty line.

  docker build -t {docker_image_name} .
  ```
  

> 2. Run the ONNXRuntime server on Docker image

Reviewer: on -> in

  ```

> 

Reviewer: Remove the empty line.

  
  docker run -it onnxruntime-ngraph
  ```

> ## nGraph Version (Preview)

Reviewer: We should have nothing related to nGraph and Python Binding.

Author: Fixed

#### Linux 16.04

> 1. Build the docker image from the workspace that contains our server "onnxruntime_server"

Reviewer: workspace -> folder or directory

Author: fixed

  ```

>   # If you have a Linux machine, preface this command with "sudo"

Reviewer: I don't think we need "sudo".

Reviewer: Same for other commands

Author: Removed
